static/js/containers/UserPage_test.js

```diff
@@ -77,13 +78,64 @@ describe("UserPage", function() {
           getElementsByClassName('delete-button')[0];
       };
 
+      beforeEach(() => {
+        let userProfile = Object.assign({}, USER_PROFILE_RESPONSE, {
+          username: SETTINGS.username
+        });
+        userProfile.education.push({
+          "id": 3,
+          "degree_name": DOCTORATE,
+          "graduation_date": "2015-12-01",
+          "field_of_study": "Philosophy",
+          "school_name": "Harvard",
+          "school_city": "Cambridge",
+          "school_state_or_territory": "US-MA",
+          "school_country": "US",
+          "online_degree": false
+        });
+        helper.profileGetStub.
+          withArgs(SETTINGS.username).
+          returns(Promise.resolve(userProfile));
+      });
+
       it('shows the education component', () => {
         return renderComponent(`/users/${SETTINGS.username}`, userActions).then(([, div]) => {
           let title = div.getElementsByClassName('profile-card-title')[1];
           assert.equal(title.textContent, 'Education');
         });
       });
 
+      it('should show the entries in resume order', () => {
```

Comment:
in my experience, repetition in test code carries the exact same problems as repetition in 'real' app code. if nothing else, splitting some of these lines into functions gives us the opportunity to express what a block of code is doing via the function name. if done right, that would make it easier to read for the uninitiated. as this reads right now, i would need to run this with console.log lines/breakpoints to figure out what it's doing and why. probably not essential for all of our test code to be immediately transparent upon first read, but i think we move a little more in that direction here